contracts/Arbitrable.sol

```diff
@@ -17,10 +17,14 @@ abstract contract Arbitrable is IArbitrable {
     bytes public arbitratorExtraData; // Extra data to require particular dispute and appeal behaviour.
 
     modifier onlyArbitrator() {
-        require(msg.sender == address(arbitrator), "Can only be called by the arbitrator.");
+        _checkOwner();
```

Comment:
I see here a really low gain of 12 gas in deployment. I think it doesn't worth the reduce of code lisibility